picard/script.py

```diff
@@ -307,6 +308,24 @@ def register_script_function(function, name=None, eval_args=True,
     )
 
 
+_int_operators = {'add': operator.add,
```

Comment:
You don't need _int_operators, you can just pass the function directly to _compute_int.